simplified-main/src/main/java/org/nypl/simplified/main/MainActivity.kt

```diff
@@ -236,29 +228,6 @@ class MainActivity :
     }
   }
 
-  /**
-   * Shows age gate for verification
-   */
-  private fun showAgeGate() {
-    ageGateDialog = AgeGateDialog()
-    ageGateDialog.show(supportFragmentManager, AgeGateDialog.TAG)
-  }
-
-  /**
-   * Handle birth year sent back from Age Gate dialog
-   */
-  override fun onBirthYearSelected(isOver13: Boolean) {
-    if (isOver13) {
-      this.profilesController.profileUpdate { description ->
-        this.synthesizeDateOfBirthDescription(description, 14)
-      }
-    } else {
-      this.profilesController.profileUpdate { description ->
-        this.synthesizeDateOfBirthDescription(description, 0)
-      }
-    }
-  }
-
   private fun synthesizeDateOfBirthDescription(
```

Comment:
I think the following two methods are not used any more.

Reply:
Nice catch, removed them!